### PR TITLE
Post about Guanyang joining the project

### PR DESCRIPTION
**Pull request by hlapp**

### CODE REVIEW

File: _posts/blog/2017-05-05-welcoming-guanyang.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+A few months ago we [opened the search]({% post_url 2016-10-05-postdoc2-ad %}) for the second[^postdocs]
+postdoc to join our project, a role which is expected to liaise with
+large NSF-funded clade-oriented research projects (such as in
+particular [GoLife] projects), and those developing informatics
+infrastructure for comparative trait analysis (such as in particular
+[ARBOR]). This search has recently concluded, and we are delighted
+to welcome Dr. [Guanyang Zhang] to our team.
+
+Guanyang is a taxonomist and an evolutionary biologist. He obtained a
```
Comment (GZhang2) [inline _posts/blog/2017-05-05-welcoming-guanyang.md]: Suggested wording: Guanyang is a taxonomist and an entomologist.

File: _posts/blog/2017-05-05-welcoming-guanyang.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+where he worked in [Christiane Weirauch's lab] on the taxonomy,
+phylogenetics, and evolution of assassin bugs (Reduviidae). After
+graduating from UCR, he moved to Arizona State University to work as a
+postdoctoral researcher with [Nico Franz]. His postdoctoral research
+revolves around the phylogenetics and evolution of weevils, but his
+interests also expanded to bacterial endosymbionts, weevil-host plant
+associations, taxonomic concepts and environmental ontology.
+
+Throughout his research, Guanyang found an imminent need to augment
+the precision and universality when communicating taxonomic concepts,
```
Comment (GZhang2) [inline _posts/blog/2017-05-05-welcoming-guanyang.md]: ... augment the precision and universality *of taxonomy for communicating and referencing biological information.*

File: _posts/blog/2017-05-05-welcoming-guanyang.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+which is why he felt the phyloreferencing project as a natural fit for
```
Comment (GZhang2) [inline _posts/blog/2017-05-05-welcoming-guanyang.md]: .. *is* a natural fit to his training and research objectives.

File: _posts/blog/2017-05-05-welcoming-guanyang.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+expanding his training. He will be particularly interested in
```
Comment (GZhang2) [inline _posts/blog/2017-05-05-welcoming-guanyang.md]: ... particularly *involved* in ..

File: _posts/blog/2017-05-05-welcoming-guanyang.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+developing and testing testing phyloreferences for insects and bacteria.
+
+Guanyang already participated in the [face-to-face project meeting]({%
```
Comment (GZhang2) [inline _posts/blog/2017-05-05-welcoming-guanyang.md]: delete *already*

**Comment by hlapp** [comment]
@GZhang2 I made the changes as requested. OK to publish?

**Comment by GZhang2** [comment]
Thanks Hilmar. I was just going to do the edit myself. Took me a while to
find out how to edit. It looks good to go. Thanks!!

Guanyang

Guanyang Zhang, PhD
Postdoc Research Associate @ The Franz Lab
School of Life Sciences, Arizona State University
PO Box 874501 (mail) Room# LSA129, 427 East Tyler Mall (package)
Tempe, AZ 85287
somanyinsects.org
Email: gyz151@gmail.com
Tel: 480-965-2850 (office) 812-320-2619 (mobile)

**Comment by gaurav** [comment]
This isn't up at http://www.phyloref.org/blog/ -- I think because it was technically published in the future (May 5) when the update was made. We could add in an extra, empty commit to trigger a rebuild: http://stackoverflow.com/a/24099328/27310

**Comment by hlapp** [comment]
I squashed the two commits into one and force pushed, but as suspected that didn't change anything. The future dated-post published fine locally, so it would have been a surprise if it didn't on Github.

The problem is that the site build fails at Github, but the only error message we're getting is this:
```
Page build failed. For more information, see https://help.github.com/articles/troubleshooting-github-pages-builds/.
```
I've now updated the Jekyll container image I'm using for local deployment (see `Dockerfile` and `serve-local.sh`), and now I am indeed getting an error:
```
Deprecation: A call to '{{ post_url 2016-07-20-welcoming-gaurav }}' did not match a post using the new matching method of checking name (path-date-slug) equality. Please make sure that you change this tag to match the post's name exactly.
       Deprecation: A call to '{{ post_url 2016-10-05-postdoc2-ad }}' did not match a post using the new matching method of checking name (path-date-slug) equality. Please make sure that you change this tag to match the post's name exactly.
  Liquid Exception: Could not find post "2017-04-26-first-face-to-face-project-meeting" in tag 'post_url'. Make sure the post exists and the name is correct. in /srv/jekyll/_posts/blog/2017-05-05-welcoming-guanyang.md
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Could not find post "2017-04-26-first-face-to-face-project-meeting" in tag 'post_url'. Make sure the post exists and the name is correct.
```
I'll try and fix that.

**Pull request merged**